### PR TITLE
HCAP-1259: footer for Employer Portal

### PR DESCRIPTION
**Pull request by alva-fresh**
https://freshworks.atlassian.net/browse/HCAP-1259

Adding a footer to the `Page` component

Screen Shot 2022-09-01 at 12 35 08 PM
Screen Shot 2022-09-01 at 12 34 57 PM
Screen Shot 2022-09-01 at 12 34 36 PM

### CODE REVIEW

**Review by fw-noel — APPROVED** [review]
With a little bit of clean up :) Love it

**Review by taraepp — APPROVED** [review]

File: client/src/components/generic/Page.js [hunk 1]
```diff
@@ -1,26 +1,32 @@
...
 const useStyles = makeStyles(() => ({
   root: (props) => ({
-    height: props.isAutoHeight ? 'auto' : 'calc(100vh - 82px)',
+    height: 'calc(100vh - 124px)',
     justifyContent: props.centered ? 'center' : 'flex-start',
     alignItems: 'center',
     flexWrap: 'nowrap',
     flexDirection: 'column',
+    display: 'flex',
+    overflow: 'scroll',
```
Comment (arranfw) [inline client/src/components/generic/Page.js:19]: We'll want `overflowY` so we don't get a persistent horizontal scrollbar throughout the site

Comment (arranfw) [reply client/src/components/generic/Page.js:19]: We might want to revisit which element handles overflow, on the participant table we end up with extra scrollbars and most aren't necessary 
image

**Review by arranfw — CHANGES_REQUESTED** [review]

File: client/src/components/generic/Page.js (hunk 1 above)
Comment (alva-fresh) [reply client/src/components/generic/Page.js:19]: Totally! I'll look more into it today, but I'm also happy to chat on how to handle this best

**Review by arranfw — APPROVED** [review]

**Review by fw-noel — APPROVED** [review]

**Pull request merged**